src/lighteval/utils/imports.py

```diff
@@ -93,3 +95,51 @@ def can_load_extended_tasks() -> bool:
 
 
 CANNOT_USE_EXTENDED_TASKS_MSG = "If you want to use extended_tasks, make sure you installed their dependencies using `pip install -e .[extended_tasks]`."
+
+
+# TODO: Might be nice to merge this with functions above
+def check_required_dependencies(step_name: str, required_dependencies: list[str] | list[tuple[str, str]]):
+    missing_dependencies: dict[str, str] = {}
+    for dependency in required_dependencies:
+        dependency = dependency if isinstance(dependency, tuple) else (dependency, dependency)
+        package_name, pip_name = dependency
+        if not _is_package_available(package_name):
+            missing_dependencies[package_name] = pip_name
+    if missing_dependencies:
+        _raise_error_for_missing_dependencies(step_name, missing_dependencies)
+
+
+def _raise_error_for_missing_dependencies(step_name: str, dependencies: dict[str, str]) -> NoReturn:
```

Comment:
```suggestion
def _raise_error_for_missing_dependencies(step_name: str, dependencies: dict[str, str])
```
Any reason to use this here ?

Reply:
No, I was really lazy and copied this stuff from datatrove